src/main/java/handlers/OperationTypeHandler.java

```diff
@@ -0,0 +1,7 @@
+package handlers;
+
+import model.FruitTransaction;
+
+public interface OperationTypeHandler {
+    void handler(FruitTransaction fruitTransaction);

```

Comment:
method names should be verbs
```suggestion
    void handle(FruitTransaction fruitTransaction);

```